### PR TITLE
CASMTRIAGE-6300 Adding work around mentioning during sat platform services 

### PR DESCRIPTION
**Pull request by shivaprasad-metimath**
IM:CASMTRIAGE-6300
Reviewer:Ryan Haskeen

while running sat bootsys boot --stage platform-services, a traceback error is observed due to instant calls in the code.

The addition of the step which checks for cronjobs that need to be recreated immediately after kubelet is started on all Kubernetes nodes. Hence adding a note in the doc as workaround to re run the cmd and validate, until a permanent fix is added.

# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by haasken-hpe — APPROVED** [review]

**Comment by haasken-hpe** [comment]
This should be backported to `release/1.5` as well

File: operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md [hunk 1]
```diff
@@ -238,6 +238,9 @@ Verify that the Lustre file system is available from the management cluster.
 
     Once Ceph is healthy, repeat the `sat bootsys boot --stage platform-services` command to finish
     starting the Kubernetes cluster.
+
```
Comment (haasken-hpe) [inline operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md]: The style checker doesn't like this line with a space at the end. Remove the trailing space.

File: operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md [hunk 2]
```diff
@@ -238,6 +238,8 @@ Verify that the Lustre file system is available from the management cluster.
 
     Once Ceph is healthy, repeat the `sat bootsys boot --stage platform-services` command to finish
     starting the Kubernetes cluster.
+    If any other errors are observed run the `sat bootsys boot --stage platform-services` command again
```
Comment (haasken-hpe) [inline operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md:241]: Just for future reference, I was not stating that the entire blank line had to be removed to satisfy the style checker, but rather that the trailing whitespace on the blank line had to be removed. That is, you can still have a blank line between lines 240 and 241, but it cannot contain any space characters.

I don't have a strong opinion on whether this needs to be a separate paragraph, so I'm approving it regardless.

**Review by haasken-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**